app/components/ProviderCards/BrightidCard.tsx

```diff
@@ -0,0 +1,224 @@
+    const res = await fetch(
+      `${process.env.NEXT_PUBLIC_DPOPP_PROCEDURE_URL?.replace(/\/*?$/, "")}/brightid/verifyContextId`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify({
+          contextIdData: userDid,
+        }),
+      }
+    );
+    const result: BrightIdProcedureReturn = await res.json();
+    return result.valid;
+  }
+
+  const handleUserVerify = (): void => {
+    if (credentialResponse) {
+      handleAddStamp(credentialResponse);
+    }
+    onClose();
+  };
+
+  // Widget displays steps to verify BrightID with Gitcoin
+  const brightIdSponsorshipWidget = (
+    <div>
+      <h1 className="text-center text-xl font-bold">BrightID</h1>
+      {userDid ? (
+        <div>
+          <br />
+          <h1>A verifiable credential was not generated for your address. Follow the steps below to qualify:</h1>
+          <br />
+          <div>
+            <p>
+              1) Download the BrightID App on your mobile device{" "}
+              <a className="text-purple-400 underline" href="https://brightid.gitbook.io/brightid/getting-started">
+                Get Started
+              </a>
+              .
+            </p>
+            <br />
+            <div>
+              2) Connect BrightID to Gitcoin by scanning this QR code from the BrightID app, or clicking{" "}
+              <a
+                className="text-purple-400 underline"
+                href={`https://app.brightid.org/link-verification/http:%2f%2fnode.brightid.org/Gitcoin/${encodeURIComponent(
+                  userDid
+                )}`}
+              >
+                here
+              </a>{" "}
+              from your mobile device.
+              <br />
+              <br />
+              <div style={{ height: "auto", margin: "auto", maxWidth: 300, width: "100%" }}>
+                <QRCode
+                  size={256}
+                  style={{ height: "auto", maxWidth: "100%", width: "100%" }}
+                  value={`brightid://link-verification/http:%2f%2fnode.brightid.org/Gitcoin/${encodeURIComponent(
+                    userDid
+                  )}`}
+                />
+              </div>
+            </div>
+
+            <Button
+              data-testid="button-sponsor-brightid"
+              width={"auto"}
+              rounded={"base"}
+              colorScheme="purple"
+              mr={3}
+              float="right"
+              onClick={async () => await handleSponsorship()}
+            >
+              <span className="font-miriam-libre">Connect BrightID</span>
+            </Button>
+          </div>
+        </div>
+      ) : (
+        <div>Refresh Browser and Try Again</div>
+      )}
+    </div>
+  );
+
+  const issueCredentialWidget = (
+    <>
+      <button
+        data-testid="button-verify-brightid"
+        className="verify-btn"
+        onClick={async () => {
+          SetCredentialResponse(undefined);
```

Comment:
might need to reset `brightIdVerification` state variable as well, since `brightIdVerification` and `credentialResponse` both come from the `fetchVerifiableCredential` response